NosTests/NoteParserTests.swift

```diff
@@ -4,6 +4,67 @@ import Dependencies
 
 final class NoteNoteParserTests: CoreDataTestCase {
 
+    func testMentionPrecededByAt() throws {
```

Comment:
in most of our tests we call the `subject` `sut` short for system-under-test. I kind of like `subject` better but let's stick with one or the other.

Reply:
I always prefer to avoid jargon; someone who's not very familiar with unit testing can probably understand `subject` more easily than `sut`, so that gets my vote.

Reply:
I renamed `expectedContent` to just `expected` and added `// Arrange`, `// Act`, and `// Assert`.

Regarding to `subject`, I always prefer **not** having static functions where possible, so if you are ok with, I can refactor `NoteParser` in the other tickets I'm working on, so we can have a `subject` here.

Reply:
I'm with you -- I prefer not to have static funcs.